packages/ee/lib/service.ts

```diff
@@ -50,14 +53,14 @@ export const getIsEnterpriseEdition = async (): Promise<boolean> => {
   const hashedKey = hashString(ENTERPRISE_LICENSE_KEY);
```

Comment:
Could we also move this `hashedKey` declaration to the top of the file and use them in all functions instead of passing it down as a prop since it stays the same on runtime?

Reply:
updated 🚀